liveRecorder/serviceWrappers/linux/liveRecorder

```diff
@@ -29,8 +29,8 @@ SCRIPT_DIR=$(dirname $(readlink -f $0))
 APPDIR=${SCRIPT_DIR}/../../
 NAME=${APP_NAME}
 APPLICATION_PATH=${APPDIR}/main.py
-LOG_DIR="/var/log/${APP_NAME}"
-PID_DIR="/var/run"
+LOG_DIR="@LOG_DIR@/${APP_NAME}"
```

Comment:
QA should and usually do use the same packages I release.
As for SaaS - Guy E. should add the replacement of tokens in his deployment script or else, he can push his own version of this script to the saas-config repo, that's what it is there for.
@GElkayam - your thoughts please.